server.js

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+import bodyParser from "body-parser";
+import fs from "fs";
```

Comment:
import `express`, the express `bodyParser` middleware, and the `fs` library for interacting with the server file system.